frontend/src/app/commonComponents/Dropdown.jsx

```diff
@@ -1,29 +1,35 @@
 import React from "react";
 import PropTypes from "prop-types";
 import useUniqueIds from "../commonComponents/useUniqueIds";
-import classnames from "classnames";
+// import classnames from "classnames";
 
 const Dropdown = ({
   options,
   label,
   name,
   onChange,
   disabled,
-  addClass,
+  // addClass,
+  id,
   defaultOption, // value of the default option
   selectedValue,
 }) => {
   const [selectId] = useUniqueIds("dropdown", 1);
+  const elemId = id || selectId;
+
+  const labelElem = label ? (
+    <label className="usa-label" htmlFor={elemId}>
+      <strong>{label}</strong>
+    </label>
+  ) : null;
 
   return (
-    <div className={classnames("prime-dropdown", addClass)}>
-      <label className="usa-label" htmlFor={selectId}>
-        {label}
-      </label>
+    <React.Fragment>
+      {labelElem}
```

Comment:
Is the label being added elsewhere? A form control without a label (or something similar like aria-label or aria-describedby) isn't 508 compliant.

Reply:
Devices list currently doesn't have a label as well since these appear in a table. I added `aria-describedby` and pointed it to the column heading.